src/write/cfi.rs

```diff
@@ -378,7 +378,6 @@ impl FrameDescriptionEntry {
 ///
 /// This may be a CFA definition, a register rule, or some other directive.
 #[derive(Debug, Clone, PartialEq, Eq, Hash)]
-#[non_exhaustive]
```

Comment:
Why remove it on the writer side too? You should only need to create `write::CallFrameInstruction` values as user of gimli, not match on them.